api/src/main/java/org/eclipse/microprofile/metrics/Metadata.java

```diff
@@ -103,7 +103,20 @@
      * </p>
      */
     private String unit = MetricUnits.NONE;
-    
+
+    /**
+     * Can this metric name (in a scope) be used multiple times?
+     * <p>
+     *   Setting this is optional. The default is <tt>false</tt>, which
+     *   prevents reusing.
+     * </p>
+     * Note that this only has an effect if the <tt>name</tt> is explicitly given.
+     *
+     * If the name is automatically determined, then this flag has no effect as
+     * all metric names are different anyway
+     */
```

Comment:
If someone sets the absolute attribute to true, then the method name is used without any package or class name prefix... so wouldn't reusable also have an effect in that case for colliding metric names based on identical method names across classes/packages?